api/providers/WarehouseProvider.cs

```diff
@@ -86,9 +89,44 @@ public List<Warehouse> GetAll()
 
     public Warehouse? GetById(Guid id) => _db.Warehouses.FirstOrDefault(l => l.Id == id);
 
-    public Warehouse? Update<IDTO>(Guid id, IDTO dto)
+    public override Warehouse? Update(Guid id, BaseDTO updatedValues)
     {
-        throw new NotImplementedException();
+        bool hasChanges = false;
```

Comment:
Deze haschanges moet weg dit is gebruikt voor een PATCH methode en aangezien dit PUT is is deze methode met haschanges niet correct.

Reply:
Zie Update Shipment provider als voorbeeld